v2.1/configure-replication-zones.md

```diff
@@ -826,70 +497,88 @@ There's no need to make zone configuration changes; by default, the cluster is c
     $ cockroach init --insecure --host=<any node hostname>
     ~~~
 
-2. On any node, configure the default replication zone:
+2. On any node, open the [built-in SQL client](use-the-built-in-sql-client.html):
 
     {% include copy-clipboard.html %}
     ~~~ shell
-    echo 'num_replicas: 5' | cockroach zone set .default \
-    --insecure \
-    --host=<any node hostname> \
-    -f -
+    $ cockroach sql --insecure
     ~~~
 
+3. Configure the default replication zone:
+
+    {% include copy-clipboard.html %}
+    ~~~ sql
+    > ALTER RANGE default CONFIGURE ZONE USING num_replicas = 5;
     ~~~
-    range_min_bytes: 1048576
-    range_max_bytes: 67108864
-    gc:
-     ttlseconds: 86400
-    num_replicas: 5
-    constraints: []
+
+4. View the replication zone:
+
+    {% include copy-clipboard.html %}
+    ~~~ sql
+    > SHOW ZONE CONFIGURATION FOR RANGE default;
+    ~~~
+    ~~~
+      zone_name |                config_sql
+    +-----------+------------------------------------------+
+      .default  | ALTER RANGE default CONFIGURE ZONE USING
+                |     range_min_bytes = 1048576,
+                |     range_max_bytes = 67108864,
+                |     gc.ttlseconds = 90000,
+                |     num_replicas = 5,
+                |     constraints = '[]',
+                |     lease_preferences = '[]'
+    (1 row)
     ~~~
 
     All data in the cluster will be replicated 5 times, including both SQL data and the internal system data.
 
-3. On any node, configure the `.meta` replication zone:
+5. Configure the `.meta` replication zone:
 
     {% include copy-clipboard.html %}
-   ~~~ shell
-   echo 'num_replicas: 7' | cockroach zone set .meta \
-   --insecure \
-   --host=<any node hostname> \
-   -f -
-   ~~~
-
-   ~~~
-   range_min_bytes: 1048576
-   range_max_bytes: 67108864
-   gc:
-     ttlseconds: 86400
-   num_replicas: 7
-   constraints: []
-   ~~~
+    ~~~ sql
+    > ALTER RANGE meta CONFIGURE ZONE USING num_replicas = 7;
+    ~~~
+
+    ~~~
+      zone_name |              config_sql
+    +-----------+---------------------------------------+
+      .meta     | ALTER RANGE meta CONFIGURE ZONE USING
+                |     range_min_bytes = 1048576,
+                |     range_max_bytes = 67108864,
+                |     gc.ttlseconds = 3600,
+                |     num_replicas = 7,
+                |     constraints = '[]',
+                |     lease_preferences = '[]'
+    (1 row)
+    ~~~
 
    The `.meta` addressing ranges will be replicated such that one copy is in all 7 datacenters, while all other data will be replicated 5 times.
 
-4. On any node, configure the `.timeseries` replication zone:
+6. Configure the `.timeseries` replication zone:
```

Comment:
This one is rendering as "1. Configure ..." #markdownfail #itsnotyourfault #computers